interfaces/cython/cantera/examples/reactors/ic_engine.py

```diff
@@ -1,18 +1,20 @@
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
```

Comment:
Since Cantera 2.5.0 and up will only support Python 3, you can remove this line.